rel-eng/build-packages-for-obs.sh

```diff
@@ -103,7 +103,7 @@ while read PKG_NAME PKG_VER PKG_DIR; do
   eval $(awk '/^Wrote:.*src.rpm/{srpm=$2}/^Wrote:.*.changes/{changes=$2}END{ printf "SRPM=\"%s\"\n",srpm; printf "CHANGES=\"%s\"\n",changes; }' "$T_LOG")
   if [ "$(head -n1 ${CHANGES}|grep '^- ')" != "" ]; then
     echo "*** Untagged package, adding fake header..."
-    sed -i "1i Fri Jan 01 00:00:00 CEST 2100 - faketagger@suse.inet\n" ${CHANGES}
+    sed -i "1i Fri Jan 01 00:00:00 CEST 2099 - faketagger@suse.inet\n" ${CHANGES}
```

Comment:
It seems that at least i586 `rpm` for SLE11 and RES6 doesn't like `2099`. Trying `2050`, and otherwise `2038`.

Reply:
Confirmed, some old `rpm` builds have the Year 2038 problem. I will change this date.